generators/new_collateral/types/collateral.go

```diff
@@ -54,9 +59,10 @@ func (c Collateral) GetFlipContractName() string {
 }
 
 func (c Collateral) GetMedianContractName() string {
-	// example: MEDIAN_ETH_B
+	// example: MEDIAN_ETH_B_1_1_3
 	name := strings.ToUpper(strings.Replace(c.Name, "-", "_", -1))
-	return fmt.Sprintf("MEDIAN_%s", name)
+	version := fmt.Sprintf("%s", strings.Replace(c.Version, ".", "_", -1))
```

Comment:
This change should be adding a version number delimited with "_" to the end of the median contract name, since we have more than one median contract versions now. This piece should be changing the version that the user enters into the script from "." delimited to "_" delimited so that we don't run into issues with other pieces of the code. If the user enters the version in with "_", then i think this line shouldn't make any changes.